ui/de-lib/src/main/java/org/iplantc/de/client/DEClientConstants.properties

```diff
@@ -2,4 +2,7 @@ fileUploadServlet = /de/secured/fileUpload
 fileDownloadServlet = /de/secured/fileDownload
 logoutUrl = j_spring_security_logout
 deHelpFile = https://pods.iplantcollaborative.org/wiki/display/DEmanual/
+notificationWS = /de/websocket/notifications
+sysmessageWS = /de/websocket/system-messages
+wsProtocol = ws://
```

Comment:
@dennisroberts71 suggested inspecting current protocol (http vs https) and then deciding between ws and wss